site/content/docs/03-template-syntax.md

```diff
@@ -149,6 +149,14 @@ Text can also contain JavaScript expressions:
 <div>{(/^[A-Za-z ]+$/).test(value) ? x : y}</div>
 ```
 
+---
+
+To include regular curly braces in the HTML, they must either be included as string Javascript expressions or their [HTML Entity](https://developer.mozilla.org/docs/Glossary/Entity) strings used
+
+```html
+{'{'} in curly braces {'}'} or &lbrace; in curly braces &rbrace; or &lcub; in curly braces &rcub;
```

Comment:
IMO we should recommend the html entity strings before js expressions as they would be the preferable choice - I feel like people tend to just use the first thing they see :sweat_smile: 

wdyt?

Reply:
`{'{'}` is a results in a bunch of overhead in the compiled code. We should definitely just be encouraging people to only use the HTML escaped version.

In a sidenote way back in the Svelte 2 blog post (https://svelte.dev/blog/version-2) we mention using `&#123;` for `{` which by chance is really easy to remember, probably moreso than the named entities in this case. If you do this, you don't need to then also escape the `}`, so you don't need to worry about that.

Reply:
:+1:  yeah sounds valid. Will change it to put the entities first. Could just not include the javascript expression if it does result in overhead?

Ah yeah, could add that as an example as well. I guess that is a case for named entities as well?

```diff
+ To include regular curly braces in the HTML, use their [HTML Entity](https://developer.mozilla.org/docs/Glossary/Entity) strings
+ 
+ ```html
+ &#123; in curly braces &#123; &lbrace; in curly braces &rbrace; or &lcub; in curly braces &rcub;
+ &#123; it also works only escaping the start curly brace }
```

Reply:
Updated in PR :smiley: